client/state/application-passwords/reducer.js

```diff
@@ -14,9 +14,10 @@ import {
 	APPLICATION_PASSWORD_NEW_CLEAR,
 	APPLICATION_PASSWORDS_RECEIVE,
 } from 'state/action-types';
-import { combineReducers } from 'state/utils';
+import { combineReducers, withSchemaValidation } from 'state/utils';
+import { itemsSchema } from './schema';
 
-export const items = ( state = [], action ) => {
+export const items = withSchemaValidation( itemsSchema, ( state = [], action ) => {
```

Comment:
There seems to be some fragmentation in the way we do this and our docs don't seem to be up to date or especially clear.

You can maintain the existing `items` definition and set `items.schema = itemsSchema`, and `combineReducers` will take care of the rest (it calls `withSchemaValidation` internally).

It seems like _most_ of our persistence uses the `reducer.schema = schema` form– a quick look suggests around 25 vs around 12.

There shouldn't be any harm doing it this way other than fragmentation.

Reply:
👍 done in https://github.com/Automattic/wp-calypso/pull/23909/commits/74b7d9fd52462fb2e2e4eaa169135b551e1ffbcd